spec/features/catalog_spec.rb

```diff
@@ -135,3 +135,26 @@
     page.response_headers['Content-Type'].should include 'application/json'
   end
 end
+
+feature 'Visitor wants to search with single or double quote' do
+  scenario 'should see the result page' do
+    visit catalog_index_path( {:q => 'fish'} )
```

Comment:
I don't see any hits for fish in my local repo -- so this test is failing.  Did you make changes to your local sample data?